include/alpaka/dev/DevCudaRt.hpp

```diff
@@ -13,6 +13,22 @@ namespace alpaka
 {
     //! The CUDA RT device handle.
     using DevCudaRt = DevUniformCudaHipRt<ApiCudaRt>;
+
+    namespace trait
+    {
+        //! The CUDA RT preferred device warp size get trait specialization.
+        template<>
+        struct GetPreferredWarpSize<DevUniformCudaHipRt<ApiCudaRt>>
+        {
+            ALPAKA_FN_HOST static auto getPreferredWarpSize(DevUniformCudaHipRt<ApiCudaRt> const& /* dev */)
+                -> std::size_t
+            {
+                // Same value for all cuda architectures
```

Comment:
I think the old comment was more descriptive:
```suggestion
                // All CUDA GPUs to date have a warp size of 32 threads.
```